anthos-attached-clusters/kind/README.md

```diff
@@ -0,0 +1,55 @@
+# Sandbox Example to Attach a [kind](https://kind.sigs.k8s.io/) Cluster using Terraform
+
+## Prerequisites
+The sample is meant just to provide a local example for experimentation. It assumes an environment where [`kind`](https://kind.sigs.k8s.io/) is available and could otherwise be run on the command line, e.g. `kind create cluster`.
+
+## A note on providers
+
+The other examples and module limit dependanices to terraform core providers, but this example takes advantage of some community supplied [providers](provider.tf). They're widely used for their purpose, but please review and consider.
+
+## Usage
+
+1. Edit the values in the terraform.tfvars file to suit your needs. Descriptions for each variable
+  can be found in `variables.tf`. Additional optional features are also available and commented out
+  in the `google_container_attached_cluster` resource in `main.tf`.
+
+    If you modify the cluster creation, ensure it meets
+  [Cluster Prerequisites](https://cloud.google.com/anthos/clusters/docs/multi-cloud/attached/eks/reference/cluster-prerequisites).
+1. Initialize Terraform:
+    ```bash
+    terraform init
+    ```
+1. Create and apply the plan:
+    ```bash
+    terraform apply
+    ```
+    Terraform may give a warning along the lines of `Warning: Content-Type is not recognized as a text type, got "application/jwk-set+json"` but this is ok and just a side effect of the `http` provider we're using and the content type the cluster returns for the `jwks` content.
+1. The process should take about a few short minutes to complete.
+1. Set some variables based on the terraform porjects values and use them to generate RBAC for the cluster and credentials to login:
+    ```bash
+
+    PROJECT=$(echo google_container_attached_cluster.primary.project | terraform console | tr -d '"')
+    CLUSTER=$(echo google_container_attached_cluster.primary.name | terraform console | tr -d '"')
+    KUBECONFIG=$(echo kind_cluster.cluster.kubeconfig_path | terraform console | tr -d '"')
+
+    # set this to whomever you'd like to grant access
+    PRINCIPAL=update.this@example.com
+    # set this whatever role you intend
+    ROLE=clusterrole/cluster-admin
+
+    gcloud container fleet memberships generate-gateway-rbac --apply \
+           --kubeconfig ${KUBECONFIG} --context=kind-${CLUSTER} \
+           --project=${PROJECT} \
+           --membership=${CLUSTER} \
+           --role=${ROLE} \
+           --users=${PRINCIPAL}
+           
+    gcloud container fleet  memberships get-credentials ${CLUSTER}  --project ${PROJECT} 
+    
+    kubectl get ns
+
+    ```
+    This will allow you to access the cluster using kubectl as you would other GKE Enterprise clusters, regardless of location (ie in GCP, other clouds, or on prem).
```

Comment:
For this quickstart example, I think using [admin_users](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_attached_cluster#admin_users) to connect to the cluster is easier?

Reply:
yes, definitely agree since this is kind and it is by definition at someone's desk. my intent was just to underscore the fact that we have a unified mechanism with the assumption that someone doing this is testing out attached for real usage on AKS or EKS.  if the overhead outweighs the message, then point taken!

Reply:
I am going to resolve with this rationale. if we want to tweak, we definitely can in another targeted PR to make all the examples have the same approach.